geopyspark/geotrellis/__init__.py

```diff
@@ -539,6 +539,48 @@ def __new__(cls, num_partitions=None, bits=8):
         return super(cls, SpatialPartitionStrategy).__new__(cls, num_partitions, bits)
 
 
+class SpaceTimePartitionStrategy(namedtuple("SpaceTimePartitionStrategy", "time_unit num_partitions bits time_resolution")):
+    """Represents a partitioning strategy for a layer that uses GeoPySpark's ``SpaceTimePartitioner``
+    with a set number of partitions, units of time, and temporal resolution.
+
+    This partitioner will try and group ``Tile``\s together that are spatially and temproally near each other
+    in the same partition. In order to do this, each ``Tile`` has their ``Key Index`` calculated
+    using the space filling curve index, ``Z-Curve``.
+
+    Note:
+        This partitiong strategy will only work on ``SPACETIME`` layers, and will fail if given a ``SPATIAL``
+        one. For ``SPATIAL`` layers, please see :class:`~geopyspark.geotrellis.SpatialPartitionStrategy`.
+
+    Args:
+        time_unit (str or :class:`~geopyspark.geotrellis.constants.TimeUnit`): Which time
+            unit should be used when saving spatial-temporal data. This controls the resolution of
+            each index. Meaning, what time intervals are used to seperate each record.
+        num_partitions (int, optional): The number of partitions that should be used during
+            partitioning. Default is, ``None``. If ``None`` the resulting layer will have
+            a ``HashPartitioner`` with the number of partitions being either the same
+            as the input layer's, or a number computed by the method.
+        bits (int, optional): How many bits the resulting ``Key Index`` of a ``Tile`` should
```

Comment:
This is a pretty inside-baseball kind of parameter.  It's crying out for more and better documentation.

Reply:
Yeah, now that I know more about I'll add a bit more to the docs to make things a little clearer.